pkg/rewards/claim.go

```diff
@@ -31,8 +32,11 @@ import (
 	"github.com/ethereum/go-ethereum/ethclient"
 
 	"github.com/urfave/cli/v2"
+	"github.com/wk8/go-ordered-map/v2"
 )
 
+const zero_address = "0x0000000000000000000000000000000000000000"
```

Comment:
should use this https://github.com/Layr-Labs/eigenlayer-cli/blob/master/pkg/utils/constants.go#L25

Reply:
also just for future. Go variables should never have `_` - idiomatic is CamelCase